src/components/Match/MatchLog.jsx

```diff
@@ -89,6 +118,14 @@ class MatchLog extends React.Component {
         <Checkbox defaultChecked label={checkbox.label} onCheck={(event, isChecked) => this.handleCheck(checkbox.label, isChecked)} />
       </span>))}
       <Table data={generateLog(this.props.match, this.state)} columns={logColumns} />
+      <div>{Object.keys(strings)
+        .filter(str => str.indexOf('rune_') === 0)
+        .map(str => str.split('_')[1])
```

Comment:
I don't think the split is necessary here, you can just use it to get runeString directly?